tests/platform_tests/api/test_chassis.py

```diff
@@ -364,22 +365,49 @@ def test_sfps(self, duthost, localhost, platform_api_conn):
 
     def test_status_led(self, duthost, localhost, platform_api_conn):
         # TODO: Get a platform-specific list of available colors for the status LED
-        LED_COLOR_LIST = [
-            "off",
-            "red",
+
+        FAULT_LED_COLOR_LIST = [
             "amber",
-            "green",
+            "red",
+            "orange"
```

Comment:
Why do we need both `amber` and `orange`? They are essentially the same color.

Reply:
Fixed.